CriteoPublisherSdk/Sources/Configuration/CR_IntegrationRegistry.m

```diff
@@ -80,4 +81,21 @@ - (BOOL)isAdMobMediationPresent {
          NSProtocolFromString(@"GADCustomEventBanner") != nil;
 }
 
+- (CR_CacheAdUnitArray *)filterAdUnits:(CR_CacheAdUnitArray *)units {
+  return [units
+      filteredArrayUsingPredicate:[NSPredicate predicateWithBlock:^BOOL(id _Nullable adUnit,
+                                                                        NSDictionary<NSString *, id>
+                                                                            *_Nullable bindings) {
+        if ([adUnit adUnitType] != CRAdUnitTypeRewarded) {
```

Comment:
In my opinion, listing exclusion is more easy that listing inclusion. For instance:

```
if (adunit is rewarded and profileId != GAM) {
  log("warning")
  return false;
}

// maybe a future exclusion ?

return true;
```

Moreover, it will then be much easier to add other exclusions then.